packages/story-editor/src/output/element.js

```diff
@@ -20,21 +20,21 @@
 import { generatePatternStyles } from '@googleforcreators/patterns';
 import { getBox } from '@googleforcreators/units';
 import { StoryAnimation } from '@googleforcreators/animation';
+import { getDefinitionForType } from '@googleforcreators/elements';
+import { BACKGROUND_TEXT_MODE } from '@googleforcreators/design-system';
+import {
+  OutputWithMask as WithMask,
+  getBorderPositionCSS,
+  getBorderRadius,
+  getBorderStyle,
+  shouldDisplayBorder,
```

Comment:
If we're moving some other element props related utils to `element-library`, maybe most of the border utils would also be better there? Most of these are not directly related to masks, as far as I know, and are mainly just creating border CSS. It's only the ones detecting if the element supports border at all that are related to masks. Thoughts?

Reply:
yes, it looks suitable for `element-library` or `elements`. The only problem is that it will create circular dependencies.

1. If we move to `elements`:
- `elements` will have `masks` dependency, because `shouldDisplayBorder` is using utils from `masks` package.
- `masks` will have to add `elements` dependency because `shouldDisplayBorder` is being used in `masks` package.

2. if we move to `element-library`:
- `element-library` is already using `masks` as a dependency.
- `masks` will have to add `element-library` as a dependency because `shouldDisplayBorder` is using utils from `masks` package.

Reply:
Hm, it looks like `shouldDisplayBorder` might need adjusting then. It currently checks for the following:
`hasBorder`, `canMaskHaveBorder`, `canSupportMultiBorder`. Maybe it needs to be split up better then so that all the relevant parts would be in the relevant packages? Or maybe this util doesn't make sense as it is anymore at all.

Could be for a follow-up issue, this PR is already huge, thoughts?

Reply:
👍 Yes, I will try in a follow-up PR.

Reply:
@sayedtaqui Looks like this still needs a follow-up issue to be created.

Reply:
I missed this one. Created here https://github.com/GoogleForCreators/web-stories-wp/issues/10917 thank you!